scripts/reversim-calendar/.gitignore

```diff
@@ -0,0 +1,176 @@
+
+# Created by https://www.gitignore.io/api/node,jetbrains
```

Comment:
That's a bit over-verbose IMO. 
We have a `.gitignore` file at the root of the project, can you please add just the entries you think are necessary to that file?